OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift

```diff
@@ -158,6 +158,20 @@ public func isMethodDELETE() -> OHHTTPStubsTestBlock {
 }
 
 /**
+ * Matcher for testing an `NSURLRequest`'s **absolute url**.
+ *
+ * e.g. the absolute url is `https://api.example.com/signin` in `https://api.example.com/signin`
+ *
+ * - Parameter scheme: The absolute url to match
+ *
+ * - Returns: a matcher (OHHTTPStubsTestBlock) that succeeds only if the request
+ *            has the given absolute url
+ */
+public func isAbsoluteUrl(_ url: String) -> OHHTTPStubsTestBlock {
```

Comment:
Totally. We tried to name most if not all our matchers from the name of the method we call underneath to make the relationship clear, use common vocabulary and avoid confusion so let's try to continue that way 😉